quora-service/src/main/java/com/upgrad/quora/service/entity/UserEntity.java

```diff
@@ -27,19 +28,19 @@
 
     @Column(name = "uuid")
     @NotNull
-    private UUID uuid;
+    private String uuid;
 
-    @Column(name = "first_name")
+    @Column(name = "firstname")
```

Comment:
Database naming must be ```first_name``` as per naming conventions and better readability. Please change for columns: first_name, last_name, user_name, about_me, contact_number

Reply:
_ is removed because it directly related to DB table users. it should match with table column name otherwise it will throw error.